packages/middleware/_package.json

```diff
@@ -1,3 +1,9 @@
+/**
+  WARNING: Package deprecated (from development).
+  Do not provide any changes here.
+  Please check the https://github.com/vuestorefront/middleware repository,
+  where the current middleware package lives.
+*/
```

Comment:
It's not a proper json file anymore. We shouldn't put comments here

Reply:
No parser will handle it

Reply:
But why, you should not release it or parse it? That's the point.